rosbag2_transport/include/rosbag2_transport/record_options.hpp

```diff
@@ -41,6 +42,8 @@ struct RecordOptions
   uint64_t compression_threads = 0;
   std::unordered_map<std::string, rclcpp::QoS> topic_qos_profile_overrides{};
   bool include_hidden_topics = false;
+  bool start_paused = false;
+  KeyboardHandler::KeyCode pause_resume_toggle_key = KeyboardHandler::KeyCode::SPACE;
```

Comment:
Though it doesn't make sense to keep it as a variable inside `RecordOptions` if we are not going to allow for users to change it. To avoid confusion and for the sake of the clarity let's remove it from the options for a while.  
I think it will be no problem to add it if we will decide to add feature for allowing to change pause key binding via `RecordOptions` and command line interface. There are no request for this and I think everybody will be happy if it will be a space bar for pause/resume.

Reply:
Fixed in https://github.com/ros2/rosbag2/pull/905/commits/1f404b9e822976af4bf4e496c613ef2538f5a01a.